libr/util/graph.c

```diff
@@ -316,6 +328,23 @@ R_API RGraphNodeInfo *r_graph_create_node_info(char *title, char *body, ut64 off
 		data->title = title;
 		data->body = body;
 		data->offset = offset;
+	} else {
+		free (title);
```

Comment:
same here. Do we really want these free here? Maybe I'm wrong, but I think the "ownership" should be transferred only when everything goes right. If something goes wrong the objects still belong to the user, who can still do whatever it wants with those data.